jwst/tweakreg/tweakreg_step.py

```diff
@@ -250,17 +250,35 @@ def process(self, input):
                 output_name = 'fit_{}_ref.ecsv'.format(self.gaia_catalog.lower())
             else:
                 output_name = None
-            ref_cat = amutils.create_astrometric_catalog(images,
-                                                         self.gaia_catalog,
-                                                         output=output_name)
+
+            self.gaia_catalog = self.gaia_catalog.strip()
+            gaia_cat_name = self.gaia_catalog.upper()
+
+            if gaia_cat_name in ['GAIADR2', 'GAIADR1']:
```

Comment:
Actually, for HST, we are already using 'GAIAeDR3' as the most up-to-date catalog.  It may be easier to maintain/update, if the list of supported pre-defined catalogs were defined as a module level (global?) variable that gets referenced here.  That makes it more obvious what catalogs are supported and adding to that list in one place becomes easier when new catalogs are supported/used.